tests/inference/qnn_test.py

```diff
@@ -475,41 +496,127 @@ def generate_resolvers():
       } for r in bitstring_resolvers]
 
     # hamiltonian energy
-    num_layers = 5
+    num_layers = 1
```

Comment:
jfme: why decrease to a single layer? to make the test faster?

Reply:
Yes, was to make the tests go faster.  Same with number of units, since the finite difference derivatives are what eat most of the time now.  Not too bad to bump it to 2 so I did that.